packages/loader/container-loader/package.json

```diff
@@ -11,17 +11,17 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
-	"main": "dist/index.js",
-	"module": "lib/index.js",
+	"main": "dist/index.cjs",
```

Comment:
Not adding an exports field yet because there are internal imports in test-end-to-end-tests. All the packages that e2e tests import from will need to start using exports at the same time, I think.